modules/ossm-rn-fixed-issues.adoc

```diff
@@ -14,7 +14,30 @@ Provide the following info for each issue if possible:
 *Result* - How has the behavior changed as a result? Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
 ////
 
-The following issue has been resolved in the current release:
+The following issues have been resolved in the current release:
+
+* https://issues.redhat.com/browse/OSSM-1397[OSSM-1397] Previously, if you removed the `maistra.io/member-of` label from a namespace, the {SMProductShortName} Operator did not not automatically reapply the label to the namespace. As a result, sidecar injection did not work in the namespace.
++
+The Operator would reapply the label to the namespace when you made changes to the `ServiceMeshMember` object, which triggered the reconciliation of this member object.
++
+Now, any change to the namespace also triggers the member object reconciliation.
+
+* https://issues.redhat.com/browse/OSSM-4851[OSSM-4851] Previously, when you attempted to create a new pod in a namespace that was part of a service mesh, the {SMProductShortName} Operator failed to create the pod. The Operator generated the following error message:
```

Comment:
This makes it sound like the error always happened. We should clarify under what circumstances it happened or say that it happened sometimes.

Reply:
@luksa do we know the circumstances in which it occurred?

Reply:
It looks like the original issue was: "Pods are not created when using the annotation "sidecar.istio.io/interceptionMode" with value "TPROXY"  

This is what we should be mentioning in the release notes, not templates or nil values, as those are an implementation detail that users are not aware of.

Reply:
Hi @luksa, in some Slack discussion we talked about not mentioning TPROXY because it's not officially supported. That is why we talk decided to talk about the nil value

Reply:
Okay, not mentioning TPROXY makes sense, but why are we even mentioning this issue at all? Saying that there was a bug in one of our templates has no value for the users. It just makes the release note confusing (even I didn't know what we were trying to say here). 

I suggest either not mentioning this fix at all, or giving something value like "In <some corner cases>, a workload pod failed to be created with the following error message: ... This issue is now fixed." (or something along those lines).

Reply:
@fjglira just wanted to circle back on this. If TPROXY is not officially supported, and @luksa pointed out users don't typically deal with helm charts or template files, do you think we should remove this from the release notes? Removing it is pretty simple to do, but before I do, I just wanted to check in and see what you think.

Reply:
ok, for me to remove it from the release notes

Reply:
Thank you Francisco. Removed the entry per group discussion during review.